### PR TITLE
Added selectize.

### PR DESCRIPTION
**Pull request by jweakley**
@Samsinite, can you take a look at this?

### CODE REVIEW

File: addon/components/bootstrap-selectize.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  isSrOnly: false,
+  selectOnTab: true,
+  classNameBindings: ['hasSuccess:has-success', 'hasWarning:has-warning', 'hasError:has-error '],
+  plugins: 'remove_button',
+  labelColumns: 2,
+  inputColumns: 10,
+
+  classNames: ['bootstrap-selectize-component', 'form-group'],
+
+  createItem: computedActionKey('create-item'),
```
Comment (Samsinite) [inline addon/components/bootstrap-selectize.js]: Looks like you need to throw the computedActionKey decorator into a util file and import it.

**Comment by Samsinite** [comment]
This should really be either tested in an app first or have an acceptance test written for it to make sure it works before merging.

**Comment by jweakley** [comment]
@Samsinite, can you point me to an example for writing an acceptance test for an ember addon?

**Comment by Samsinite** [comment]
Yup. See https://github.com/wildland/ember-sanctify/blob/master/tests/acceptance/index-route-access-test.js and the dummy section https://github.com/wildland/ember-sanctify/tree/master/tests/dummy/app

**Comment by Samsinite** [comment]
Also see http://alisdair.mcdiarmid.org/2015/06/20/ember-component-integration-tests.html for component integration tests that we can write now.

**Comment by Samsinite** [comment]
Not ready to merge, reopen when ready

**Pull request closed without merging**